packages/date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -52,6 +58,16 @@ export const DatePickerMixin = (subclass) =>
           value: '',
         },
 
+        /**
+         * Date used as the basis to parse short dates.
+         * @type {Date | undefined}
+         * @protected
+         */
+        _referenceDate: {
+          type: Date,
```

Comment:
Shouldn't the reference date also be a string in ISO 8601 format like the rest of the date properties on the Web Component API (`value`, `initialPosition`, `min`, `max`)? A Date object is tricky to deal with correctly due to timezone issues.

Reply:
Converted `Date` into `DatePickerDate` in the latest commit. That object is consistent with what we use with `parseDate` and `formatDate` functions within `i18n`.

Reply:
I think the idea was rather to have a string property with a format like `yyyy-mm-dd`, like the one here:
https://github.com/vaadin/web-components/blob/375b23de7c8bde67de116b888c53b8a54ca6acea/packages/date-picker/src/vaadin-date-picker-mixin.js#L242-L253

While `parseDate` and `formatDate` work with `DatePickerDate` that is more of an internal thing. For configuring the date picker all other properties use the string format, and it's also more convenient to use than having to pass an object.

Reply:
Addressed in the latest commit